Find a functionally equivalent code:android.view.ViewGroup.unFocus:COMMENT
Method Modifier: 
Comment:/**
 * {@inheritDoc}
 */

@@ -3,11 +3,11 @@
         System.out.println(this + " unFocus()");
     }
     if (mFocused == null) {
-        super.unFocus();
+        super.unFocus(focused);
     } else {
-        mFocused.unFocus();
+        mFocused.unFocus(focused);
         mFocused = null;
     }
 }

